src/components/App.js

```diff
@@ -32,12 +32,16 @@ class App extends Component {
     this.props.listenToQueries();
   }
 
+  removePod = (query) => {
```

Comment:
this notation is used to bind this implicitly (see https://medium.com/@housecor/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56#.tpfyexc3z, option 5)